ros_buildfarm/templates/snippet/builder_shell.xml.em

```diff
@@ -1,3 +1,4 @@
     <hudson.tasks.Shell>
       <command>@ESCAPE(script)</command>
+      <configuredLocalRules/>
```

Comment:
* hudson.tasks.Shell added the ability to filter or reset environment
variables from specific steps and the default configuration saves a
self-closing configuredLocalRules tag which contains that information.